scss/components/_watermark.scss

```diff
@@ -4,6 +4,7 @@
   max-width: 100%;
   pointer-events: auto;
   outline: none !important;
+  z-index: $zindex-watermark;
```

Comment:
Looks good overall, but please make sure to check the following:

- Are there any other elements affected by the higher z-index of `.oo-player-container` that we haven't found yet? This was also affecting the up next panel, so there could be others remaining.
- Does the new z-index for watermark place it on top of other elements that it shouldn't be on top of? Not sure if there's a case where this is true, but we should verify.